File: Nível 1/1933 - Tri-du/main.lua
valor = io.read()

x1, x2 = valor:match("(%S+)%s+(%S+)")
x1, x2 = tonumber(x1), tonumber(x2)

if x1 > x2 then
    print(x1)
elseif x2 > x1 then
    print(x2)
elseif x1 == x2 then
    print(x1)
else
    print("Erro")
end

File: Nível 1/1933 - Tri-du/main.py
valor = input("A ")

x1,x2 = valor.split(" ")

x1 = int(x1)
x2 = int(x2)

if x1 > x2:
    print(x1)
elif x2 > x1:
    print(x2)
elif x1 == x2:
    print(x1)
else:
    print("Erro")